packages/z/zonos-elements.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "zonos-elements",
+  "filename": "dist/scripts/loadZonos.js",
+  "description": "Zonos component UI for Stripe elements",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/types/index.d.ts",
+  "unpkg": "dist/zonos-elements/zonos-elements.esm.js",
+  "collection": "dist/collection/collection-manifest.json",
```

Comment:
```suggestion
```
Please remove these properties, as cdnjs does not support/expect them in a library JSON config.

Reply:
Thanks for the suggestion.